src/ui/i18n/i18n.js

```diff
@@ -0,0 +1,136 @@
+import path from 'path';
```

Comment:
I don't think do. Index.js shouldn't contain code in my opinion, just the exported bindings for a directory.